src/coreComponents/physicsSolvers/wavePropagation/sem/acoustic/secondOrderEqn/isotropic/AcousticWaveEquationSEM.cpp

```diff
@@ -510,8 +578,10 @@ real64 AcousticWaveEquationSEM::computeTimeStep( real64 & dtOut )
 
     stiffnessVector.zero();
     p.zero();
+    //Lien to ensure that the using array stays on GPU (useful when we cal this routine several times)
+    forAll< parallelHostPolicy >( sizeNode, [p] ( localIndex const ){} );
```

Comment:
Should probably be replaced by a "touch" on CPU

Reply:
After discussion, let's keep it like this for now, as it guarantees that corner cases involving `pyGEOS` scripts work as expected.